composer.json

```diff
@@ -15,28 +15,59 @@
   "scripts": {
     "test:unit": "./vendor/bin/phpunit --color",
     "test:typing": "./vendor/bin/phpstan analyse",
-    "test:syntax": "./vendor/bin/php-cs-fixer fix ./src --dry-run --stop-on-violation --using-cache=no"
+    "test:syntax": "./vendor/bin/php-cs-fixer fix --dry-run --stop-on-violation --using-cache=no",
+    "test:benchmark": "./vendor/bin/phpbench run -l dots --report aggregate"
   },
   "require": {
-    "php": ">=7.2",
-    "ext-curl": "*",
+    "php": ">=7.4",
     "ext-json": "*",
-    "ext-mbstring": "*",
-    "ext-openssl": "*",
-    "guzzlehttp/guzzle": "^7.0.1|^6.2",
-    "web-token/jwt-signature": "^2.0",
-    "web-token/jwt-key-mgmt": "^2.0",
-    "web-token/jwt-signature-algorithm-ecdsa": "^2.0",
-    "web-token/jwt-util-ecc": "^2.0"
+    "beberlei/assert": "^3.2",
+    "psr/cache": "^1.0",
+    "psr/event-dispatcher": "^1.0",
+    "psr/http-client": "^1.0",
+    "psr/http-factory": "^1.0",
+    "psr/http-message": "^1.0",
+    "psr/log": "^1.1",
+    "symfony/var-dumper": "^5.1",
```

Comment:
VarDumper should be in require-dev?

Reply:
Hi,

Actually, it will be removed when the PR is finished.